pytrainer/lib/date.py

```diff
@@ -22,34 +22,71 @@
 import datetime
 import calendar
 import dateutil.parser
-from dateutil.tz import * # for tzutc()
+from dateutil.tz import tzutc, tzlocal
 import logging
 
+def second2time(seconds):
+    if not seconds:
+        return 0,0,0
+    hours = seconds // (60*60)
+    seconds %= (60*60)
+    minutes = seconds // 60
+    seconds %= 60
+    return hours,minutes,seconds
+
+def time2second(time):
+    hour,min,sec = time
+    return int(sec)+(int(min)*60)+(int(hour)*3600)
+
+def getLocalTZ():
+    ''' Returns string representation of local timezone'''
+    return datetime.datetime.now(tzlocal()).tzname()
+
+def time2string(date):
+    return "%0.4d-%0.2d-%0.2d" %(int(date[0]),int(date[1]),int(date[2]))
+
+def getNameMonth(date):
+    day, daysInMonth = calendar.monthrange(date.year, date.month)
+    monthName = calendar.month_name[date.month]
+    return monthName, daysInMonth
+
+def unixtime2date(unixtime):
+    print unixtime
```

Comment:
David García Granda <notifications@github.com> writes:
>> +    return int(sec)+(int(min)*60)+(int(hour)*3600)
>> +
>> +def getLocalTZ():
>> +    ''' Returns string representation of local timezone'''
>> +    return datetime.datetime.now(tzlocal()).tzname()
>> +
>> +def time2string(date):
>> +    return "%0.4d-%0.2d-%0.2d" %(int(date[0]),int(date[1]),int(date[2]))
>> +
>> +def getNameMonth(date):
>> +    day, daysInMonth = calendar.monthrange(date.year, date.month)
>> +    monthName = calendar.month_name[date.month]
>> +    return monthName, daysInMonth
>> +
>> +def unixtime2date(unixtime):
>> +    print unixtime
>
> Would not be a good ides to get rid of *print* statement in
> *unixtime2date*?

Sure, why not. I have pushed the change.

So far in this series I had not modified any of the code in lib/date.py,
only reorganized it. There are also commented out debug prints in
getDateTime, should those also be removed or perhaps converted to
logging.debug?

-- 
Arto Jantunen